src/fs_storage.rs

```diff
@@ -52,18 +53,32 @@ impl Storage for FsStorage {
     }
 
     fn entries(&self, low: u64, high: u64, _max_size: u64) -> Result<Vec<Entry>, raft::Error> {
-        let mut entries: Vec<Entry> = read_entries_sorted_by_index(&self.entries_dir)?;
+        if low > high {
+            return err_compacted();
+        }
+
+        let first_entry_index = read_first_index(&self.data_dir)?;
+        let last_entry_index = read_last_index(&self.data_dir)?;
+
+        println!("low={}, high={}, first_entry_index={}, last_entry_index={}", low, high, first_entry_index, last_entry_index);
```

Comment:
Do you want this println rather than a log statement?

Reply:
Accidentally left that in from debugging